src/content-helper/editor-sidebar/smart-linking/smart-linking.tsx

```diff
@@ -122,4 +125,24 @@ export const initSmartLinking = (): void => {
 	 * Initialize the block overlay component.
 	 */
 	initBlockOverlay();
+
+	/**
+	 * If the smart-link query parameter is present, select the smart link in the editor.
+	 * This is used to highlight the smart link in the editor when the user clicks on a smart link
+	 * in the review modal, on a different page.
```

Comment:
```suggestion
	/**
	 * If the smart-link query parameter is present, select the smart link in the editor.
	 *
	 * This is used to highlight the smart link in the editor when the user clicks on a smart link
	 * in the review modal, on a different page.
```

I'm having a difficulty grasping the "different page". Wondering if there's a better way we can explain this.

Reply:
Yeah, I agree! I will try to rewrite it a little bit better.